spec/classes/qpid_client_spec.rb

```diff
@@ -11,35 +11,35 @@
         it { is_expected.to compile.with_all_deps }
         it { is_expected.to create_package('qpid-cpp-client-devel') }
 
-        it 'should create configuration file' do
+        it 'creates configuration file' do
           verify_exact_contents(catalogue, '/etc/qpid/qpidc.conf', [
-            'log-enable=error+',
-          ])
+                                  'log-enable=error+',
+                                ],)
         end
       end
 
       context 'with ssl options' do
         let :params do
           {
-            :ssl                     => true,
-            :ssl_port                => 5671,
-            :ssl_cert_db             => "/etc/pki/katello/nssdb",
-            :ssl_cert_password_file  => "/etc/pki/katello/nssdb/nss_db_password-file",
-            :ssl_cert_name           => "broker",
+            ssl: true,
+            ssl_port: 5671,
+            ssl_cert_db: '/etc/pki/katello/nssdb',
+            ssl_cert_password_file: '/etc/pki/katello/nssdb/nss_db_password-file',
+            ssl_cert_name: 'broker',
           }
         end
 
         it { is_expected.to compile.with_all_deps }
         it { is_expected.to create_package('qpid-cpp-client-devel') }
 
-        it 'should create configuration file' do
+        it 'creates configuration file' do
           verify_exact_contents(catalogue, '/etc/qpid/qpidc.conf', [
-            'log-enable=error+',
-            'ssl-port=5671',
-            'ssl-cert-db=/etc/pki/katello/nssdb',
-            'ssl-cert-password-file=/etc/pki/katello/nssdb/nss_db_password-file',
-            'ssl-cert-name=broker'
-          ])
+                                  'log-enable=error+',
+                                  'ssl-port=5671',
+                                  'ssl-cert-db=/etc/pki/katello/nssdb',
+                                  'ssl-cert-password-file=/etc/pki/katello/nssdb/nss_db_password-file',
+                                  'ssl-cert-name=broker'
+                                ],)
```

Comment:
I know these are Rubocop rules, but I always disliked how these end up formatting with massive indents when it comes to readability. Just a comment, no action needed.

Reply:
Perhaps this is a rule we want to change. The trailing comma is also a bit weird.